sql/src/main/java/io/crate/jobs/JobContextService.java

```diff
@@ -127,6 +128,15 @@ public JobExecutionContext createOrMergeContext(JobExecutionContext.Builder cont
         }
     }
 
+    public void killAll() {
+        Iterator<Map.Entry<UUID, JobExecutionContext>> iterator = activeContexts.entrySet().iterator();
```

Comment:
think we must do some locking here to prevent creating new contexts while a killAll is running...